File: backend/database/models/Review.cs
using System.ComponentModel.DataAnnotations;
using backend.database.models;
using backend.Models.Base;

namespace backend.Models;

public class Review : BaseEntity
{
    [MaxLength(2000)]
    public required string Text { get; set; }
    public required int Score { get; set; }
    
    public required Guid UserId { get; set; }
    public User User { get; set; }
    
    public required Guid EventId { get; set; }
    public Event Event { get; set; }
}

File: frontend/src/viewEvent/Review.ts
export class Review {
    public readonly reviewerNickname: string;
    public readonly score: number;
    public readonly text: string;
    
    constructor(
        reviewerNickname: string,
        score: number,
        text: string,
    ) {
        this.reviewerNickname = reviewerNickname;
        this.score = score;
        this.text = text;
    }
}